client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -2,29 +2,46 @@
  * External dependencies
  */
 import React from 'react';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import GoMobile from 'my-sites/customer-home/cards/features/go-mobile';
 import ChecklistSiteSetup from 'my-sites/customer-home/cards/primary/checklist-site-setup';
 import MasteringGutenberg from 'my-sites/customer-home/cards/education/mastering-gutenberg';
-import QuickLinks from 'my-sites/customer-home/cards/primary/quick-links';
+import GrowEarn from 'my-sites/customer-home/cards/features/grow-earn';
+import Stats from 'my-sites/customer-home/cards/features/stats';
+import FreePhotoLibrary from 'my-sites/customer-home/cards/education/free-photo-library';
 
 const cardComponents = {
-	'home-feature-go-mobile-phones': GoMobile,
+	'home-feature-stats': Stats,
+	'home-feature-grow-and-earn': GrowEarn,
 	'home-primary-checklist-site-setup': ChecklistSiteSetup,
-	'home-primary-quick-links': QuickLinks,
 	'home-education-mastering-gutenberg': MasteringGutenberg,
+	'home-education-free-photo-library': FreePhotoLibrary,
 };
 
 const Primary = ( { checklistMode, cards } ) => {
+	const translate = useTranslate();
+
+	if ( ! cards ) {
+		return null;
+	}
+
 	// Always ensure we have primary content.
-	if ( cards && cards.length < 1 ) {
-		cards = [ 'home-primary-quick-links' ];
+	if ( cards.length < 1 ) {
+		cards = [ 'home-feature-stats' ];
 	}
+
+	const stats = cards.shift();
+
 	return (
 		<div className="primary">
+			<h2>{ translate( 'Stats at a glance' ) }</h2>
+			{ React.createElement( cardComponents[ stats ], {
+				checklistMode: stats === 'home-primary-checklist-site-setup' ? checklistMode : null,
+			} ) }
+			<h2>{ translate( 'Learn and grow' ) }</h2>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Learn and grow with My Home' )` *ES Score: 6*